apps/snitch_core/test/data/model/payment/card_payment_test.exs

```diff
@@ -9,6 +9,7 @@ defmodule Snitch.Data.Model.CardPaymentMethodTest do
   setup :user_with_address
   setup :an_order
   setup :payment_methods
+  setup :card
   setup :card_payment
 
```

Comment:
Please add a test to check if we can update the `card` used in some `card_payment`

Reply:
Sure

Reply:
No need for this.